corehq/apps/app_manager/models.py

```diff
@@ -5635,9 +5635,10 @@ def import_app(app_id_or_source, domain, source_properties=None, request=None):
                 try:
                     config.report_id = report_map[config.report_id]
                 except KeyError:
-                    raise AppEditingError(
-                        "Report {} not found in {}".format(config.report_id, domain)
-                    )
+                    if check_all_reports or config.report(source_domain).is_static:
```

Comment:
should this if statement be before the try block instead?

Reply:
I debated where to put it, don't think it matters either way.